### ISSUE TITLE
Thoughts on adding a basic Phoenix example?

### ISSUE THREAD

**Issue by kw7oe** [opentelemetry-beam/opentelemetry-examples#10]
Hi, I notice that we have an ongoing work on the `phoenix_backend`. However, I have not much context on what's the original plan and what's the end goal like.

I have setup a super basic Phoenix + Ecto application integrated with `opentelemetry_phoenix` and `opentelemetry_ecto` [here](https://github.com/kw7oe/phoenix-opentelemetry-example). It's based on what we have on `basic_elixir`, so we have docker compose setup and user can run and play around with it locally with ease. 

I'm thinking down the road, we could extend the example to configure `otel-collector-config.yaml` to export the traces directly to service like honeycomb and Lightstep as well.  

Do you think it would be good to include that into this repository as well?  If yes, should we replace the existing `phoenix_backend` example, or create a new folder?

**Comment by tsloughter** [comment]
Yes, that would be great, including the idea to have it include optional configuration for exporting to services like honeycomb and lightstep.

Phoenix and beam-telemetry in general seem to be where many people will start coming to Otel from with the intention of getting spans and right now I think the docs and examples don't meet them where they are but instead starts from a blank beginning.

**Comment by kw7oe** [comment]
Ah great! I'll open a PR soon to include that. Thanks.
